Source/BugsnagNotifier.m

```diff
@@ -335,10 +420,16 @@ - (void)orientationChanged:(NSNotification *)notif {
   case UIDeviceOrientationFaceDown:
     orientation = @"facedown";
     break;
-  case UIDeviceOrientationUnknown:
   default:
-    orientation = @"unknown";
+    return; // always ignore unknown breadcrumbs
+  }
+  
+  NSDictionary *lastBreadcrumb = [[self.configuration.breadcrumbs arrayValue] lastObject];
+    
+  if (lastBreadcrumb && [orientation isEqualToString:lastBreadcrumb[@"name"]]) {
```

Comment:
Good catch - I've updated this to check for the Orientation Change notification, and to inspect the breadcrumb metadata instead.